libraries/splunk_password.rb

```diff
@@ -0,0 +1,99 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: splunk_password.rb
+
+require 'base64'
+
+# Module contains different functions to encrypt and decrypt splunk passwords
+module CernerSplunk
+  # RC4 implementation taken from https://github.com/caiges/Ruby-RC4
```

Comment:
1) If we're going to copying an implementation, we need to conform to the terms of the license (MIT) in this case. 
2) This is [part of a gem](https://rubygems.org/gems/ruby-rc4) should we be using chef-gem to install and manage the version of it, instead of copying this code into our codebase?